src/main/java/org/ja13/eau/misc/Coordonate.kt

```diff
@@ -143,7 +144,22 @@ class Coordonate : INBTTReady {
     }
 
     fun distanceTo(e: Entity): Double {
-        return Math.abs(e.posX - (x + 0.5)) + Math.abs(e.posY - (y + 0.5)) + Math.abs(e.posZ - (z + 0.5))
+        //return Math.abs(e.posX - (x + 0.5)) + Math.abs(e.posY - (y + 0.5)) + Math.abs(e.posZ - (z + 0.5))
+        return Math.sqrt(Math.pow((e.posX - (x+0.5)),2.0) + Math.pow((e.posY - (y+0.5)),2.0) + Math.pow((e.posZ - (z+0.5)),2.0))
```

Comment:
This is really expensive compared to the previous function. I think we were going for speed over accuracy.

Reply:
I tested the two methods and the previous one didn't cause any regressions so I went ahead returned it back to the cheaper method.